### ISSUE TITLE
Improvement: Add method setUserAvatar to UserApi

### ISSUE THREAD

**Issue by vpreclik** [gmessner/gitlab4j-api#314]
This works for me

```java
	public User setUserAvatar(User user, File avatarFile)
			throws GitLabApiException
	{
		Response response = putUpload(Response.Status.OK, "avatar", avatarFile,
				"users", user.getId());
		return (response.readEntity(User.class));
	}
```

**Comment by gmessner** [comment]
@vpreclik 
This has been added and is available in release 4.9.18, please verify and close when you have a chance.

**Comment by gmessner** [comment]
@vpreclik 
I'm closing this since the functionality has been added and released.  Feel free to re-open if you find any issues with the implementation.
